src/org/zaproxy/zap/extension/ascanrulesBeta/CrossDomainScanner.java

```diff
@@ -35,7 +35,7 @@
 import javax.xml.xpath.*;
```

Comment:
Would fixing this wildcard import fall under the clean coders piece of the dev guidelines?